### ISSUE TITLE
Custom Heads not loading textures in inventory

### ISSUE THREAD

**Issue by alexmartz710** [PaperMC/Paper#3651]
### What behaviour is expected:
Be able to see the texture of the custom head in my inventory.

### What behaviour is observed:
When opening inventory custom head shows as a Steve head, once you place the Steve head, the texture from the custom head loads in, but not in your inventory

### Steps/models to reproduce:
Give yourself any head from minecraft-heads.com and you'll only see a Steve head in your inventory but when you place it, it gains the texture

Example command: /minecraft:give @p minecraft:player_head{display:{Name:"{\"text\":\"Demon\"}"},SkullOwner:{Id:[I;492378479,-1598995490,-1709455590,1556980441],Properties:{textures:[{Value:"eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvOTVlNGZkZmI5NjE3MzFjYmI3MTMxMWE2NzQzNDRhZGUzOWQ5MTFkM2M4ZGI4MzhkYTFjMDk5ODFhYmMzMDc1OCJ9fX0="}]}}} 1

### Plugin list:
Irrelevant

### Paper build number:
1.16.1 #8

### Anything else:
___Anything else you think may help us resolve the problem___

**Comment by MiniDigger** [comment]
Can you provide an example command please?

**Comment by alexmartz710** [comment]
`/minecraft:give @p minecraft:player_head{display:{Name:"{\"text\":\"Demon\"}"},SkullOwner:{Id:[I;492378479,-1598995490,-1709455590,1556980441],Properties:{textures:[{Value:"eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvOTVlNGZkZmI5NjE3MzFjYmI3MTMxMWE2NzQzNDRhZGUzOWQ5MTFkM2M4ZGI4MzhkYTFjMDk5ODFhYmMzMDc1OCJ9fX0="}]}}} 1`

**Comment by Thatsmusic99** [comment]
Because I believe this is related to the problem, custom heads appear to not have the textures applied to them specified in their game profile. For context, I used my plugin, HeadsPlus (v6.10.9, which added 1.16 support) to give me my own head, which uses my UUID to apply the texture - it did that fine. However, when giving myself a custom head which is generated by the plugin, it is duplicated across the whole GUI, taking the first head that appeared.

Tested using Paper-10 and compared against git-Spigot-758abbe-ca19c2f which did not have the problem.

![2020-06-28_13 04 25](https://user-images.githubusercontent.com/25277367/85947139-d91fb200-b940-11ea-8616-9e2dcd04645f.png)
![2020-06-28_13 06 25](https://user-images.githubusercontent.com/25277367/85947156-e5a40a80-b940-11ea-8b63-12503952c9aa.png)

I also found the issue occurring in one of my other plugins, SimplePets (experimental build), making me conclude that it is a Paper problem and not ours.
